src/Symfony/Component/HttpFoundation/Annotation/Stateless.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Annotation;
+
+/**
+ * Annotation class for @Stateless().
+ *
+ * @Annotation
+ * @Target({"CLASS", "METHOD"})
+ *
+ * @author Mathias Arlaud <mathias.arlaud@gmail.com>
+ */
+class Stateless
```

Comment:
As discussed, let's get rid of the annotation - configuring this with regular route config is good enough, for now at least.